Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt

```diff
@@ -109,9 +110,18 @@ class HomeFragmentViewModel @AssistedInject constructor(
                     }
                 }
             }
+            if (cwaSettings.lastAppVersion.value < BuildConfigWrap.VERSION_CODE &&
+                LocalData.isOnboarded() && LocalData.onboardingCompletedTimestamp()
```

Comment:
Because isOnboarded() is set to true right after pressing the "Next" button from the last screen in the onboarding flow. When we reach the home fragment and we perform this check, the isOnboarded() value already returns true. Luckily I've found that we also stored the timestamp of the onboarding so I've used it to avoid any changes to the existing logic.